src/main/scala/org/camunda/feel/impl/interpreter/FeelInterpreter.scala

```diff
@@ -676,14 +597,13 @@ class FeelInterpreter {
         case ValYearMonthDuration(y) => ValLocalDateTime(x.plus(y))
         case ValDayTimeDuration(y)   => ValLocalDateTime(x.plus(y))
         case _ =>
-          error(y, s"expect Year-Month-/Day-Time-Duration but found '$x'")
+          error(s"expect Year-Month-/Day-Time-Duration but found '$x'")
```

Comment:
🔧 Wouldn't this be an `INVALID_TYPE` instead of `UNKNOWN`? There is a lot of these cases where the message says I expect one of these types, but I found something else. I'm pretty sure these are impossible to reach cases, but if it would occur it would be an invalid type.

Reply:
Yes, correct. I postponed these cases for later to limit the scope of this PR.